.gitignore

```diff
@@ -1 +1,5 @@
-build/
+# IDE Metadata
+.vs/
```

Comment:
Can you remove the ide specific ignores.  I think the best practice is to add this to your global gitignore file and not to pollute project specific gitignore with these directories

Reply:
IDE Metadata section is now removed from the `.gitignore`.